src/media_import/__init__.py

```diff
@@ -9,6 +13,15 @@
 try:
     import Crypto.Cipher
```

Comment:
I changed this line to this from `import Crypto.Cipher`
I'm not sure why but when running the add-on with the previous line there was no exception (I uninstalled the package and cleared the lib folder before running it). It definetely worked before and it probably didn't throw an exception because I had the package installed earlier but this method of checking if the package is installed seems to be more reliable when testing.